js/slider.js

```diff
@@ -0,0 +1,51 @@
+
+const sliderPriceElement = document.querySelector('.ad-form__slider');
+const inputPrice = document.querySelector('#price');
+const selectType = document.querySelectorAll('#type');
+const minPriceRoom = {
+  bungalow: '0',
+  flat: '1000',
+  hotel: '3000',
+  house: '5000',
+  palace: '10000'
+};
+
+
+noUiSlider.create(sliderPriceElement, {
+  range: {
+    min: 1000,
+    max: 100000,
+  },
+  start: 1000,
+  step: 100,
+  connect: 'lower',
+  format: {
+    to: function (value) {
+      return value.toFixed(0);
+    },
+    from: function (value) {
+      return parseFloat(value);
+    },
+  },
+});
+
+function setValueSlider () {
+  sliderPriceElement.noUiSlider.updateOptions({
+    range: {
+      min: parseFloat(minPriceRoom[this.value]),
+      max: 100000,
+    },
+    start: parseFloat(minPriceRoom[this.value]),
```

Comment:
Предлагаю не сбрасывать цену при изменении типа жилья а оставлять слайдер в том же месте, а если цена была ниже чем минимум у слайдера преносить в минимум слайдера
```suggestion
  start: Math.max(parseFloat(minPriceRoom[this.value]), parseFloat(inputPrice.value))
```